### ISSUE TITLE
SCM - Add accessibility help dialog

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#203577]
cc @jooyoungseo, @lszomoru 

JooYoung mentioned there are some tab stops that aren't necessary here, making navigating it cumbersome. 

He also indicated that we should give users an overview of this view and inform them of useful actions that can be taken.

**Comment by meganrogge** [comment]
If possible, the changes should be tab navigable (right now a user has to use down arrow to reach them)

**Comment by jooyoungseo** [comment]
Another related issue is that there is no command that allows users to directly focus in the change tree view. I suggest the following command to be added:

1. SCM: Focus in change view

2. SCM: Focus in staged view

**Comment by meganrogge** [comment]
https://github.com/microsoft/vscode/issues/210255

**Comment by meganrogge** [comment]
consolidating issues:

SCM view needs some commands exposed because atm, you have to focus the SCM view and then tab or use space/arrow keys to get to these.

focus commit message
focus staged changes (not tabbable)
focus changes (not tabbable)
We should add !screenReaderEnabled to this keybinding so users can review their commit messages without going up in history
{ "key": "up", "command": "scm.viewPreviousCommit", "when": "scmInputIsInFirstPosition && scmRepository && !suggestWidgetVisible" }

**Comment by meganrogge** [comment]
@lszomoru had the idea to alert with the number of staged / changed files when the SCM view is focused. One issue with this is in the multi-repo case. For that case, we could instead indicate the number of repositories that are opened. 

We discussed fixing:
- https://github.com/microsoft/vscode/issues/204300 which would also fix the issue related to navigating in history, as the user knows what the input is and thus knows when they're about to go into history
- exposing commands to 1 ) Focus in change view 2. SCM: Focus in staged view
- indicate these in a new accessibility help dialog

**Comment by lszomoru** [comment]
The latest VS Code Insiders release contains commands to focus next/previous input/resource group:
* `workbench.scm.action.focusNextInput`
* `workbench.scm.action.focusPreviousInput`
* `workbench.scm.action.focusNextResourceGroup`
* `workbench.scm.action.focusPreviousResourceGroup`

@jooyoungseo, please give them a try and let me know if they are working as expected. Thanks!

**Comment by lszomoru** [comment]
@meganrogge, is there anything else outstanding on this issue or can we close it? Thanks!

**Comment by meganrogge** [comment]
just the accessibility help dialog for the SCM view

**Comment by lszomoru** [comment]
The first iteration of the accessibility help dialog will be in Monday's VS Code Insiders release.
